### PR TITLE
chore: bump WWS to 1.5.0

### PR DESCRIPTION
**Pull request by ereslibre**
Bump WWS to 1.5.0

### CODE REVIEW

**Comment by Mossaka** [comment]
Need a rebase

**Comment by ereslibre** [comment]
Smoke test working fine, this is ready for review :)

File: Makefile [hunk 1]
```diff
@@ -93,7 +93,7 @@ update-deps:
 
 test/out_%/img.tar: images/%/Dockerfile
 	mkdir -p $(@D)
-	docker buildx build --platform=wasi/wasm --load -t $(TEST_IMG_NAME_$*) ./images/$*
+	docker buildx build --provenance=false --platform=wasi/wasm --load -t $(TEST_IMG_NAME_$*) ./images/$*
```
Comment (Mossaka) [inline Makefile:96]: Could you please explain why setting provenance to false here as I am not so familiar with provenance attestations. Even better if you could write a comment explaining above this line.

**Review by Mossaka — APPROVED** [review]
Just left one comment. Mostly LGTM!

File: Makefile (hunk 1 above)
Comment (ereslibre) [reply Makefile:96]: When building with buildx on my current ubuntu-based VM, I'm getting: "ERROR: failed to solve: no match for platform in manifest...".

<details>
<summary>Complete buildx output</summary>

```
docker buildx build --platform=wasi/wasm --load -t wasmtest_wws:latest ./images/wws
[+] Building 0.4s (5/5) FINISHED                                                                                                                                                  docker:default
 => [internal] load build definition from Dockerfile                                                                                                                                        0.0s
 => => transferring dockerfile: 107B                                                                                                                                                        0.0s
 => [internal] load .dockerignore                                                                                                                                                           0.0s
 => => transferring context: 2B                                                                                                                                                             0.0s
 => [internal] load build context                                                                                                                                                           0.0s
 => => transferring context: 30B                                                                                                                                                            0.0s
 => CACHED [1/1] COPY --chmod=0755 ./hello.js .                                                                                                                                             0.0s
 => ERROR exporting to image                                                                                                                                                                0.2s
 => => exporting layers                                                                                                                                                                     0.0s
 => => exporting manifest sha256:37fecfda4b76efe65283b3e7ad60251262c6648cc4379aeed0ba8f524d7af782                                                                                           0.0s
 => => exporting config sha256:2a94bf471b8da7a262c690effbecaaddea5f5697d1986ab564f5b95989b0315d                                                                                             0.0s
 => => exporting attestation manifest sha256:2afe3934e1218e1ff3e4892d2699a5b8ac9b75e2cded3e088b7e6b9672dd0e69                                                                               0.1s
 => => exporting manifest list sha256:363816af6aeab000e88f0a8458449eff31ec63715e1149bfcc8f66c1f8d9fb40                                                                                      0.0s
 => => naming to docker.io/library/wasmtest_wws:latest                                                                                                                                      0.0s
 => => unpacking to docker.io/library/wasmtest_wws:latest                                                                                                                                   0.0s
------
 > exporting to image:
------
ERROR: failed to solve: no match for platform in manifest sha256:363816af6aeab000e88f0a8458449eff31ec63715e1149bfcc8f66c1f8d9fb40: not found
make: *** [Makefile:96: test/out_wws/img.tar] Error 1
```

</details>

This seems related to https://github.com/moby/buildkit/issues/3891. This was workarounded in https://github.com/moby/buildkit/pull/3983, released with v0.12.0-rc2 onwards. However, the version of Docker I have installed is 24.0.5, which [seems to use buildkit 0.11.7](https://github.com/moby/moby/blob/a61e2b4c9c5f7c241aeb37f389b4444aee26bea4/vendor.sum#L1046-L1047).

I'm going to update the PR with a specific commit for this change on the Makefile (while adding a comment as you suggested).

If you think we should avoid adding this completely I'm fine as well, I can keep this change on my stash of local patches :)

**Comment by ereslibre** [comment]
PR updated with an explanation on the Makefile with a dedicated commit :)

**Comment by ereslibre** [comment]
CI seems to be failing due to a transient issue with GH actions.

**Review by Mossaka — APPROVED** [review]

**Comment by Mossaka** [comment]
@ereslibre could you please sign your commits?

**Comment by ereslibre** [comment]
> @ereslibre could you please sign your commits?

I'm sorry again 🙏 I forgot 🤦 

I just updated the PR, both commits signed with GPG and signed off too.

**Review by Mossaka — APPROVED** [review]

**Comment by Mossaka** [comment]
Thank you! @ereslibre I will merge once the CI passes

**Pull request merged**